### DISCUSSION TITLE
Is there a way to download member-only videos from YouTube?

### DISCUSSION THREAD

**Discussion by KSXGitHub** [NickvisionApps/Parabolic#633] [Q&A] [answered]
I used to open the YouTube site to watch them directly, but now that YouTube has implemented anti adblock, I would like to download them to watch them on my local machine.

**Comment by nlogozzo** [comment, marked as answer]
Hi!

What do you mean by member-only videos? 

Videos that require you to be logged into YouTube to watch them?
If that is the case, you have to use cookies to allow Parabolic to access the video.
- First, open the member-only YouTube video in your browser and make sure you are logged in.
- Then, open Parabolic's preferences and scroll down till you see `Cookies File`.
- Click the help icon next to cookie file and you'll find the extension needed for your browser to get a cookies file
- Save the cookies file somewhere on your PC and go back to Parabolic's preferences and upload the cookies file there
- You should now be able to add a new download in Parabolic and simply paste the link of the video and click `Validate` and it should be able to access the member-only video for you to download.

**Reply by KSXGitHub** [reply]
Do I have to use different cookies file for different videos or just reuse the same one?

**Reply by nlogozzo** [reply]
> Do I have to use different cookies file for different videos

No. One cookies file should cover all the videos because the cookies file is what contains your encoded credentials for YouTube so the video can be accessed. 

I just say to grab the cookies file after you load the video in the browser to make sure you are signed in and the cookies file is up-to-date.

Cookie files usually expire/become invalid after a certain period of time, so if Validating a member-only video in Parabolic starts to fail after a certain period of time, it means you have to get a new cookie file and reupload it to Parabolic.
